doc/neps/nep-0006-newbugtracker.rst

```diff
@@ -1,7 +1,7 @@
 .. _NEP06:
 
 ===================================================
-NEP 6 — Replacing Trac with a different bug tracker
+NEP 6 — Replacing trac with a different bug tracker
```

Comment:
Similarly, I think Trac is a proper name and should be capitalized.

Reply:
I shall make the change. 
![Screenshot 2021-09-08 at 8 23 26 PM](https://user-images.githubusercontent.com/17350312/132533731-73148ca4-b525-4dd0-81fc-165676f4823c.png)